R/print.R

```diff
@@ -15,6 +15,7 @@ print.hardhat_blueprint <- function(x, ...) {
   cat_line("  # Outcomes: {n_blueprint_outcomes(x)}")
   cat_line("   Intercept: {x$intercept}")
   cat_line("Novel Levels: {x$allow_novel_levels}")
+  cat_line(" Composition: {x$composition}")
```

Comment:
This will print out `Composition:` for all hardhat blueprint types, not just for the recipe one (see the diffs in the output tests).

Take a look at `print.formula_blueprint`. It prints out `Indicators:` just for recipe blueprints, and uses `NextMethod()` to print out everything from the `print.hardhat_blueprint()` method first